src/TotallyOrderedBallotMarginTieBreaker.php

```diff
@@ -0,0 +1,71 @@
+<?php
+namespace PivotLibre\Tideman;
+
+use \InvalidArgumentException;
+use PivotLibre\Tideman\Ballot;
+use PivotLibre\Tideman\CandidateTest;
+use PivotLibre\Tideman\MarginTieBreaker;
+use PivotLibre\Tideman\CandidateComparator;
+use PivotLibre\Tideman\SingleBallotMarginTieBreaker;
+
+/**
+ * This class breaks ties using a simplified version of the tie breaking rule published in Tideman and Zavist's 1989
+ * paper "Complete Independence of Clones". The simplification is that this class requires that
+ * the TBRC (aka Ballot) contains no ties.
+ */
+class TotallyOrderedBallotMarginTieBreaker implements MarginTieBreaker
+{
+    private $candidateComparator;
+
+    /**
+     * Construct a MarginTieBreaker using a CandidateComparator that establishes a total ordering over all Candidates.
+     * The CandidateComparator must use a Ballot that contains no ties.
+     * In other words,the CandidateComparator should return nonzero integers for all Candidates, unless the two
+     * Candidates being compared are actually the same Candidate as determined by comparing their Id properties:
+     * ($candidateA->getId() == $candidateB->getId())
+     *
+     */
+    public function __construct(CandidateComparator $candidateComparator)
+    {
+        if ($candidateComparator->getBallot()->containsTies()) {
```

Comment:
@lucasdailey The first pass at tie-breaking requires a tie-breaking ballot that contains no ties. Is this simplification ok?

Reply:
Totally okay. What's a little random-parsing between friends.